<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleDialog</class>
 <widget class="QDialog" name="ScaleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="gbHorizontal">
     <property name="title">
      <string>Horizontal Scale</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Interval:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leHIncrement">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbHFixed">
        <property name="text">
         <string>Fixed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbVertical">
     <property name="title">
      <string>Vertical Scale</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Interval:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leVIncrement">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbVFixed">
        <property name="text">
         <string>Fixed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cbHFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>leHIncrement</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>ScaleDialog</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbVFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>leVIncrement</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
